src/applications/facility-locator/utils/mapHelpers.js

```diff
@@ -92,7 +92,7 @@ export const reverseGeocodeBox = (bounds, types = 'address,postcode') => {
 };
 
 export const staticMapURL = (lat, long, mapboxToken) =>
-  `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/pin-l+e31c3d(${long},${lat})/${long},${lat},16/500x300?access_token=${mapboxToken}`;
+  `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/pin-l+d83933(${long},${lat})/${long},${lat},16/500x300?access_token=${mapboxToken}`;
```

Comment:
is this an actual hexcode update or something different?

Reply:
I researched this and it looks like this is how you tell mapbox what color the pin should be